src/FixedDataTableCell.js

```diff
@@ -277,11 +260,26 @@ class FixedDataTableCell extends React.Component {
       );
     }
 
-    var cellProps = {
-      columnKey,
-      height,
-      width
-    };
+    var cellProps;
+    if(!this.props.isHeader) {
```

Comment:
nit: put space between if and parenthesis

Reply:
> This means FDT is well aware of the resizer component.
> Is there any way to remove/minimize this coupling?

This is just for spawning the columnResizerComponent when the user uses the old API. On removing the older API, this piece of code can be removed.